src/desktop/components/location_search/index.styl

```diff
@@ -16,41 +16,3 @@
 form.is-validated
   .google-maps-places-input:invalid
     box-shadow inset 0 0 0 2px red-color
-
-.pac-container
-  position relative
-  box-shadow none
-  border 2px solid purple-color
-  border-top 0
-  border-radius 0
-  // Google will do weird things to the z-index for the results dropdown
-  // While kind of lame, this ensures we can stick it in a modal
-  z-index zindex-modal
-  // Google logo
-  &:after
-    default-garamond-font-size
-    padding 20px 0
-    background-position 100% 50%
-    margin-right 10px
-
-.pac-item
-  position relative
-  padding 10px
-  box-sizing content-box
-  border-color gray-lighter-color
-  &:first-child
-    border-top none
-  &:last-child
-    border-bottom 1px solid gray-lighter-color
-  &:hover
-    background-color gray-lightest-color
-  &.pac-item-selected
-    border-color transparent
-    background-color #000
-    > .pac-item-query
-      color #fff
-
-.pac-item,
-.pac-item-query
-  garamond()
-  font-size default-garamond-font-size
```

Comment:
Old instances of this will just use the default Google styling, which is fine. The new component has styling that matches the rest of our autocompletes.